### DISCUSSION TITLE
More features

### DISCUSSION THREAD

**Discussion by minhdanlc124** [tejado/Authorizer#60] [Ideas]
i got the password generator from the chrome browser and i think there should be a feature to quickly enter passwords without having to enter them one by one

**Comment by tejado** [comment]
So, you would like to have an import function? The psafe3 format - that is used by Authorizer - was developed by https://pwsafe.org/. So there are applications for various platforms like Windows and macOS (https://pwsafe.org/relatedprojects.shtml) to open psafe3 files and also import from other sources. For the original client, the import feature is described here: https://pwsafe.org/help/pwsafeEN/html/import.html

Does that make sense or did I misunderstand you?
